sdk/storage/Azure.Storage.Files.Shares/api/Azure.Storage.Files.Shares.netstandard2.0.cs

```diff
@@ -788,6 +792,20 @@ public enum ShareFileRangeWriteType
         Update = 0,
         Clear = 1,
     }
+    public partial class ShareFileRenameOptions
+    {
+        public ShareFileRenameOptions() { }
+        public string ContentType { get { throw null; } set { } }
+        public Azure.Storage.Files.Shares.Models.ShareFileRequestConditions DestinationRequestConditions { get { throw null; } set { } }
+        public Azure.Storage.Files.Shares.Models.NtfsFileAttributes? FileAttributes { get { throw null; } set { } }
+        public System.DateTimeOffset? FileCreatedOn { get { throw null; } set { } }
+        public System.DateTimeOffset? FileLastWrittenOn { get { throw null; } set { } }
```

Comment:
The spec also mentions `x-ms-file-change-time`, which we are missing in this class.